telephony/android_modem.h

```diff
@@ -239,6 +239,11 @@ extern void amodem_set_gsm_location( AModem modem, int lac, int ci );
 extern int amodem_get_base_port( AModem  modem );
 extern int amodem_get_instance_id( AModem  modem );
 
+/** Enable/Disable the selected modem feature
+ **/
+
+extern int amodem_enable_disable( AModem  modem, const char*  feature, bool  enable );
```

Comment:
If I were you, I would have an enum for these features, rename this function to `amodem_enable_feature`, optionally add `amodem_has_feature`:
```C
/* in telephony/android_modem.h */
typedef enum {
    A_MODEM_FEATURE_HOLD = (0x01 << 0),
} AModemFeature;

/*in telephony/android_modem.c */
typedef struct AModemRec_
{
    uint32_t features;
} AModemRec;

int
amodem_enable_feature(AModem modem, AModemFeature feature, bool on)
{
    if (on)
        // set bit
    else
        // clear bit
}

bool
amodem_has_feature(AModem, AModemFeature)
{
    return modem->features & feature;
}

/* in android/console.c */
static const char* gsm_feature_names[] = { "call-hold" };

static int
do_gsm_enable_disable(ControlClient  client, char  *args, bool enable) { ... }
```